india_compliance/gst_india/doctype/gst_settings/gst_settings.js

```diff
@@ -115,4 +116,10 @@ function set_auto_generate_e_waybill(frm) {
         "auto_generate_e_waybill",
         frm.doc.auto_generate_e_invoice && frm.doc.generate_e_waybill_with_e_invoice
     );
+
+    frm.set_value("cancel_e_waybill", frm.doc.cancel_e_invoice)
 }
+
+function cancel_e_invoice(frm){
+    frm.set_value("cancel_e_waybill", frm.doc.cancel_e_invoice)
```

Comment:
I guess we need to ensure same reason of cancellation for both too. As for our current design this cannot be different, when we cancel both of them simultaneously.